sunpy/tests/figure_hashes_py35.json

```diff
@@ -1,29 +1,30 @@
+    "sunpy.map.tests.test_plotting.test_peek_grid_aia171": "e64690246e0abdd7c3b41b7eacdf15e5efd82ac0af02c820155b5954a3f032fd",
+    "sunpy.map.tests.test_plotting.test_peek_grid_limb_aia171": "ed2bce754a47f6b7115234c081ed9539c6a63871b97206b7df1b6aec205b9e47",
+    "sunpy.map.tests.test_plotting.test_peek_grid_spacing_aia171": "9a83ddfe4d68cb6111de7050dc8c43ba725b1a9017c784dfa0d7dc746613afdd",
+    "sunpy.map.tests.test_plotting.test_peek_limb_aia171": "621f086ce7bf1abd9c303d5e7050f63b77a80388773dbfb1fbc2206ff3aa22e4",
+    "sunpy.map.tests.test_plotting.test_plot_aia171": "0d25061fdc1ad0167ad7fb7d56ba8f895df6f4a4c7178ae49fdbd45ed54707a7",
+    "sunpy.map.tests.test_plotting.test_plot_aia171_nowcsaxes": "f0f335c59621718804227860171823408c64a4cfcb4e9451f8ffbbb0099351ce",
+    "sunpy.map.tests.test_plotting.test_plot_aia171_superpixel": "382037489f7851a4fc0d08528942d8036d3cc2183472397fa87bd22450bc1fed",
+    "sunpy.map.tests.test_plotting.test_plot_aia171_superpixel_nowcsaxes": "fb20a189bb4df2f09c5d534ab4de7b4277d72e51e1475a9f06501a10c29a1ba9",
+    "sunpy.map.tests.test_plotting.test_plot_masked_aia171": "f99fe3abd2ae93bfd599fd2861025e66ecef8f255f586b315a970fb9b9f26e19",
+    "sunpy.map.tests.test_plotting.test_plot_masked_aia171_nowcsaxes": "e853fdacec8d606fff106b3f5bcf92a07db00558a0e1f3a947022248a8e8f337",
+    "sunpy.map.tests.test_plotting.test_plot_masked_aia171_superpixel": "7422778204438942b151e6df0f59857b6d398c3f70772821f121843b5a550415",
+    "sunpy.map.tests.test_plotting.test_plot_masked_aia171_superpixel_nowcsaxes": "60532a3fbec7d151b74a83bf6e2488a764164595eeb24fc935567729d0d52dc4",
+    "sunpy.map.tests.test_plotting.test_rectangle_aia171": "a71b04a82645b4793fc97f12c1655fd101572bdfdee1d2df60a601954f1fdd01",
+    "sunpy.timeseries.tests.test_timeseriesbase.test_eve_peek": "ed39b8cfb4264b70e21e178e7bcecb5a11ea3d55c4fbd9a62a9ac71f1601d72c",
+    "sunpy.timeseries.tests.test_timeseriesbase.test_fermi_gbm_peek": "daccdd2245dba424e4f6edc55fe0beed1842d36b66912050a7b3910c7b947449",
+    "sunpy.timeseries.tests.test_timeseriesbase.test_generic_ts_peek": "69aefdc55c2a330c8d6752d9fd18b0d882da79d3f6f9fa047958217d50a80d0f",
+    "sunpy.timeseries.tests.test_timeseriesbase.test_lyra_peek": "a3ff6b9481ead6756de4c4f0f0f7eacf6d0d39e25ed553bfa6818ce71fbf0d68",
+    "sunpy.timeseries.tests.test_timeseriesbase.test_noaa_ind_peek": "dc8a633161599e0c9fb3cd6d5425d0f31b6c4e50dbe7a24e7469b3b8b8ccc742",
+    "sunpy.timeseries.tests.test_timeseriesbase.test_noaa_pre_peek": "76739eb613572901474b2f573508ff27324a6183175ee3cb43e5b5c21b54999d",
+    "sunpy.timeseries.tests.test_timeseriesbase.test_norh_peek": "f388fa1332a109404ed581464e882a24c46a51b54c9a481a57546a453b0fabfc",
+    "sunpy.timeseries.tests.test_timeseriesbase.test_rhessi_peek": "c4b9ba470cd2840790ee72ddc20f587129cfdeabb21260a225ca3d369fb4f3ab",
+    "sunpy.visualization.tests.test_mapcubeanimator.test_plot_mapcubeanimator": "4b9dcff676adfac27d34a3a08d1ddb36aa8c943e887622240dbb15d470a1ba6e"
 }
```

Comment:
Why have the hashes changed?

Reply:
This way sound weird but with these set of hashes, tests are passing locally. I don't know how?

Reply:
Are you using the fig env files we use for these tests?
Also we have two sets of hashes, for py2.7 and py3.5